### DISCUSSION TITLE
Can't seem to use ammo I create

### DISCUSSION THREAD

**Discussion by Volcac** [fvtt-fria-ligan/twilight2000-foundry-vtt#56] [Q&A]
When typing in ammo type into the the weapon I can use it and it pops up in the drop down, but the ammo I make as a separate item do not show up on the list. trying to make HE and AP shells for vehicle cannons while I wait on the core set to be available. and i cannot get these to pop up in the list nor overrule the weapon stats. does anyone know what I'm doing wrong?
![FoundryVTT_TkbC4VISaf](https://user-images.githubusercontent.com/87490378/138179332-5c2c8660-2c40-488c-beb6-05ea8ef89f6d.png)

**Comment by Stefouch** [comment]
Is it fixed now? (sorry for the late answer, I never got any notifications of this message)
